ui/apps/platform/src/Components/PatternFly/IconText/PolicyStatusIconText.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement } from 'react';
+import { ThumbsDown, ThumbsUp } from 'react-feather';
```

Comment:
Minor question: why not use the PF Icon versions instead of react-feather?

I assumed that `react-feather` would be going away once we are fully migrated to PF, but I don't know if I have any basis for that assumption...